frontend/src/components/NotificationsCentral/style.scss

```diff
@@ -0,0 +1,42 @@
+@use '../../theme/variables.scss' as vars;
+
+.#{vars.$bcgov-prefix}--row{
+  margin: 0rem;
+}
+
+.btn-viewed {
+  padding-right: 3.75rem;
+}
+
+.btn-all-notifications {
+  padding-right: 3.31rem;
+}
+
+.empty-section-container {
```

Comment:
Use other class exclusively of your changes to make this css rule more specific, e.g.: `vertical-center .empty-section-container`.